package/MDAnalysis/analysis/pca.py

```diff
@@ -217,10 +222,15 @@ def _prepare(self):
         self._n_atoms = self._atoms.n_atoms
 
         if self._mean is None:
-            self.mean = np.zeros(self._n_atoms*3)
+            self.mean = np.zeros((self._n_atoms,3))
```

Comment:
pep 8: space after comma
```suggestion
            self.mean = np.zeros((self._n_atoms, 3))
```